### PR TITLE
[5.0] Use own icon for dashboard due to fontawesome changes

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #40748 .

### Summary of Changes
An own dashboard icon was made by @angieradtke. Thank you!
I added a new folder into the atum image folder and changed the rendering for the atum submenu 

### Testing Instructions
Attention: needs npm or the package for test. 

### Actual result BEFORE applying this Pull Request
see #40748

### Expected result AFTER applying this Pull Request
The dashboard looks as before in J4.x
grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 1]
```diff
@@ -143,9 +144,23 @@
...
+    $pathDashboard = 'media/templates/administrator/atum/images/icons/dashboard.svg';
+    $attrDashboard = [
+        'loading'     => 'eager',
+        'decoding'    => 'async',
+        'aria-hidden' => 'true',
+        'class'       => 'atum-dashboard',
+        'height'      => '18px'
+    ];
+
+    $iconDashboard  = HTMLHelper::_('image', $pathDashboard, '', $attrDashboard, false, 0);
```
Comment (Quy) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
    $iconDashboard = HTMLHelper::_('image', $pathDashboard, '', $attrDashboard, false, 0);
```

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on a56ad7c6ea96c303c50e83195f89ce4538c3cb3e<br /><br />does what it says - no idea if this is the best way <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41443">issues.joomla.org/tracker/joomla-cms/41443</a>.</sub>

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 2]
```diff
@@ -143,9 +144,23 @@
...
     $titleDashboard = Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($current->title));
+
+    // Prepare the Dashboard icon. We use an own icon, not Fontawesome
+    $pathDashboard = 'media/templates/administrator/atum/images/icons/dashboard.svg';
+    $attrDashboard = [
+        'loading'     => 'eager',
+        'decoding'    => 'async',
+        'aria-hidden' => 'true',
+        'class'       => 'atum-dashboard',
+        'height'      => '18px'
```
Comment (sandewt) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
        'height'      => '18px',
```

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 3]
```diff
@@ -143,9 +144,23 @@
...
+    ];
+
+    $iconDashboard = HTMLHelper::_('image', $pathDashboard, '', $attrDashboard, false, 0);
+
     echo '<span class="menu-dashboard"><a href="'
-        . Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $current->dashboard) . '">'
-        . '<span class="icon-th-large" title="' . $titleDashboard . '" aria-hidden="true"></span>'
+        . Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $current->dashboard) . '"'
+        . 'title="'. $titleDashboard .'">'
```
Comment (sandewt) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
        . 'title="' . $titleDashboard . '">'
```

File: administrator/modules/mod_submenu/tmpl/default.php [hunk 4]
```diff
@@ -93,8 +93,23 @@
...
+                                        $titleDashboard = Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($child->title));
+
+                                        // Prepare the Dashboard icon. We use an own icon, not Fontawesome
+                                        $pathDashboard = 'media/templates/administrator/atum/images/icons/dashboard.svg';
+                                        $attrDashboard = [
+                                            'loading'     => 'eager',
+                                            'decoding'    => 'async',
+                                            'aria-hidden' => 'true',
+                                            'class'       => 'atum-dashboard',
+                                            'height'      => '18px'
```
Comment (sandewt) [inline administrator/modules/mod_submenu/tmpl/default.php]:
```suggestion
                                            'height'      => '18px',
```

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 5]
```diff
@@ -143,9 +144,23 @@
...
+        'class'       => 'atum-dashboard',
+        'height'      => '18px'
+    ];
+
+    $iconDashboard = HTMLHelper::_('image', $pathDashboard, '', $attrDashboard, false, 0);
+
     echo '<span class="menu-dashboard"><a href="'
-        . Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $current->dashboard) . '">'
-        . '<span class="icon-th-large" title="' . $titleDashboard . '" aria-hidden="true"></span>'
+        . Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $current->dashboard) . '"'
```
Comment (Quy) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
        . Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $current->dashboard) . '" '
```

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 6]
```diff
@@ -143,9 +144,23 @@
...
     $titleDashboard = Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($current->title));
+
+    // Prepare the Dashboard icon. We use an own icon, not Fontawesome
+    $pathDashboard = 'media/templates/administrator/atum/images/icons/dashboard.svg';
+    $attrDashboard = [
+        'loading'     => 'eager',
+        'decoding'    => 'async',
+        'aria-hidden' => 'true',
+        'class'       => 'atum-dashboard',
+        'height'      => '18px',
```
Comment (dgrammatiko) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]: FWIW the attributes `height` and `width` are unit-less: [HTML5 img spec](https://www.w3.org/wiki/Html/Elements/img)

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 3f7276a8d7608de15a9fecf2dd1fcedd70ae3de7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41443">issues.joomla.org/tracker/joomla-cms/41443</a>.</sub>

**Comment by ChristineWk** [comment]
pr-41443

pr-41443-2

File: administrator/modules/mod_menu/tmpl/default_submenu.php (hunk 6 above)
Comment (chmst) [reply administrator/modules/mod_menu/tmpl/default_submenu.php]: thx

Comment (Quy) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
        'height'      => '18',
```

**Comment by chmst** [comment]
Agree @Quy. I made an extra folder in images for so we can expand. Especially as the new fontawesome icons look worse than before. And woud be happy to do that for all atum icons if a desinger helps with svg.

File: administrator/modules/mod_submenu/tmpl/default.php [hunk 7]
```diff
@@ -93,8 +93,23 @@
                                 <?php endif; ?>
                                 <?php if ($item->dashboard) : ?>
                                     <span class="menu-dashboard">
-                                        <a href="<?php echo Route::_('index.php?option=com_cpanel&view=cpanel&dashboard=' . $item->dashboard); ?>">
-                                            <span class="icon-th-large" title="<?php echo htmlentities(Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($child->title))); ?>"></span>
+                                        <?php
+                                        $titleDashboard = Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($child->title));
+
+                                        // Prepare the Dashboard icon. We use an own icon, not Fontawesome
```
Comment (Quy) [inline administrator/modules/mod_submenu/tmpl/default.php]:
```suggestion
                                        // Prepare the Dashboard icon. We use our own icon, not Fontawesome
```

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 8]
```diff
@@ -143,9 +144,23 @@
 
 if (!empty($current->dashboard)) {
     $titleDashboard = Text::sprintf('MOD_MENU_DASHBOARD_LINK', Text::_($current->title));
+
+    // Prepare the Dashboard icon. We use an own icon, not Fontawesome
```
Comment (Quy) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]:
```suggestion
    // Prepare the Dashboard icon. We use our own icon, not Fontawesome
```

**Comment by HLeithner** [comment]
@angieradtke is the icon created by you? it looks extremely similar to the fontawesome 5 icon which could lead us into copyright troubles but I'm not lawyer so I have to ask...

**Comment by angieradtke** [comment]
@HLeithner  I painted this icon with illustrator and then edited the source code in php storm. This is an icon with 4 squares that have a border radius. I can't judge how big the intellectual property of these four squares is, especially since font-awesome has modified this icon and uses the customized icon to display a grid. As far as I know font awesome is Open Font under SIL

**Comment by HLeithner** [comment]
> @HLeithner I painted this icon with illustrator and then edited the source code in php storm. This is an icon with 4 squares that have a border radius. I can't judge how big the intellectual property of these four squares is, especially since font-awesome has modified this icon and uses the customized icon to display a grid. As far as I know font awesome is Open Font under SIL

perfect thanks then I can merge this.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on f271ec1ed65847d5f2c164b3617553c0a516ed10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41443">issues.joomla.org/tracker/joomla-cms/41443</a>.</sub>

**Pull request merged**